DEV-1/DEV-1/Sequence.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+namespace DEV_1
+{
+    class Sequence
+    {
+        public Sequence() { }
+        public void SequenceSearch(string word)
+        {
+            bool flag = false;//Repeating symbol indicator
+            string seq;//Keeps non-repeating symbols 
+            seq = string.Empty;
+            seq += word[0];
+            for (int i = 1; i < word.Length; i++)
+            {
+                if (word[i] != word[i - 1])
+                {
+                    seq += word[i];
+                }
+                else
+                {
+                    flag = true;
+                }
+                if (flag || (i == word.Length - 1))
+                {
+                    SequenceOutput(seq);//output of all combinations found in 'seq' 
+                    seq = string.Empty;
+                    seq += word[i];
+                    flag = false;
+                }
+            }
+        }
```

Comment:
In this method you both search sub-sequences and output them, single responsibility violation